components/badge.css

```diff
@@ -0,0 +1,52 @@
+:root {
+    --available-color: #008000;
+    --unavailable-color: #ff0000;
```

Comment:
Do better variable naming

Reply:
Sure thing anik. Thanx for the feedback.